Userland/Applications/Spreadsheet/Spreadsheet.cpp

```diff
@@ -208,7 +212,11 @@ Optional<Position> Sheet::parse_cell_name(const StringView& name) const
 
 Optional<size_t> Sheet::column_index(const StringView& column_name) const
```

Comment:
Why is this function even needed?
We know the index of a column to create the string representing its index in the first place. Whoever wants the index for a given column should be able to get it from the column directly or from whoever created the colns and keeps track of them. This route seems very round-about. 
But I guess this issue is out of scope of this pr...

Reply:
That function is needed for the JS interop side of the story : )